frontend/app/src/components/home/Home.svelte

```diff
@@ -217,11 +217,11 @@
             // TODO - deal with channels
```

Comment:
Can this comment go now?

Reply:
I want to leave it there until I've dealt with invitations which is not done yet